src/Apps/Order/redirects.tsx

```diff
@@ -186,3 +205,39 @@ export const redirects: RedirectRecord<{
     },
   ],
 }
+
+graphql`
+  fragment redirects_order on Order {
+    id
+    mode
+    state
+    ... on OfferOrder {
+      myLastOffer {
+        id
+        createdAt
+      }
+      lastOffer {
+        id
+        createdAt
```

Comment:
These `createdAt` fields don’t appear to be used anywhere in the code.

Reply:
they are used in the `goToRespondIfMyLastOfferIsNotMostRecentOffer` predicate

Reply:
Ohhh, don’t know how I missed that!